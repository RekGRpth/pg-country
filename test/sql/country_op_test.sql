BEGIN;
CREATE EXTENSION country;
SELECT 'uk'::country > 'de'::country;
SELECT 'uk'::country >= 'de'::country;
SELECT NOT('uk'::country < 'de'::country);
SELECT NOT('uk'::country <= 'de'::country);
ROLLBACK;
BEGIN;
CREATE EXTENSION country;
SELECT 'uk'::country > 'de'::country;
SELECT 'uk'::country >= 'de'::country;
SELECT NOT('uk'::country < 'de'::country);
SELECT NOT('uk'::country <= 'de'::country);
ROLLBACK;
BEGIN;
CREATE EXTENSION country;
SELECT NOT('de'::country = 'uk'::country);
SELECT 'uk'::country = 'uk'::country;
ROLLBACK;
BEGIN;
CREATE EXTENSION country;
SELECT 'de'::country != 'uk'::country;
SELECT NOT('uk'::country != 'uk'::country);
ROLLBACK;
