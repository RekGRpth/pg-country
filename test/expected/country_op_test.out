BEGIN;
CREATE EXTENSION country;
SELECT 'uk'::country > 'de'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'uk'::country >= 'de'::country;
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country < 'de'::country);
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country <= 'de'::country);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
BEGIN;
CREATE EXTENSION country;
SELECT 'uk'::country > 'de'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'uk'::country >= 'de'::country;
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country < 'de'::country);
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country <= 'de'::country);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
BEGIN;
CREATE EXTENSION country;
SELECT NOT('de'::country = 'uk'::country);
 ?column? 
----------
 t
(1 row)

SELECT 'uk'::country = 'uk'::country;
 ?column? 
----------
 t
(1 row)

ROLLBACK;
BEGIN;
CREATE EXTENSION country;
SELECT 'de'::country != 'uk'::country;
 ?column? 
----------
 t
(1 row)

SELECT NOT('uk'::country != 'uk'::country);
 ?column? 
----------
 t
(1 row)

ROLLBACK;
